tests/array.rs

```diff
@@ -2466,74 +2466,48 @@ mod array_cow_tests {
 
 #[test]
 fn test_remove_index() {
-    let mut a = arr2(&[[1, 2, 3],
-                       [4, 5, 6],
-                       [7, 8, 9],
-                       [10,11,12]]);
+    let mut a = arr2(&[[1, 2, 3], [4, 5, 6], [7, 8, 9], [10, 11, 12]]);
```

Comment:
If we can avoid it, we won't commit changes like this where nice visual formatting is removed

Reply:
I reverted the changes to the original and reapplied the fixes.